src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Building Unified Namespace using Node-RED and MQTT
+subtitle: Join Grey Dziuba, OT Data and Community Strategist, and Kudzai Manditereza, Developer Advocate at HiveMQ, as they discuss MQTT and Node-RED and how it pertains to the Unified Name Space, UNS.
+image: 
+date: 2024-03-26
+time: 17:00 CET (11:00am ET) 
+duration: 60
+# video: rl6uwV2PhOc
+hosts: ["grey-dziuba"]
+hubspot:
+    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
+---
+
+**The digital transformation journey for many manufacturers hasn’t always been a smooth one.  From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.**
+
+<!--more-->
+
```

Comment:
The digital transformation journey for many manufacturers hasn’t always been a smooth one.  From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.